### ISSUE TITLE
attribute values not added (to a semi-open list)

### ISSUE THREAD

**Issue by peterstadler** [TEIC/romajs#67]
## How to reproduce

* start with minimal schema and build up
* modify attributes of `<unclear>` by changing the `@reason` attribute (see screenshot)
  * change list from "semi-open" to "closed"
  * delete some valItems
  * add a new valItem (here: "overwritten")
* Download customization

![Bildschirmfoto 2019-09-11 um 13 07 31](https://user-images.githubusercontent.com/2247936/64692549-b7c70580-d495-11e9-816b-731ac715070f.png)

## Resulting ODD snippet

```xml
<elementSpec ident="unclear" mode="change">
  <attList>
    <attDef ident="reason" mode="change">
      <valList type="closed" mode="change">
        <valItem mode="change" ident="overwritten"/>
        <valItem mode="delete" ident="inaudible"/>
        <valItem mode="delete" ident="faded"/>
        <valItem mode="delete" ident="background_noise"/>
        <valItem mode="delete" ident="eccentric_ductus"/>
      </valList>
    </attDef>
  </attList>
</elementSpec>
```

## Problem 

The `<valItem mode="change" ident="overwritten"/>`  is changed, not added, and is not available in the resulting RelaxNG schema.

## What did I expect

The new value should be added, like

```xml
<valItem mode="add" ident="overwritten"/>
```

**Comment by raffazizzi** [comment]
Fixed in 0.2.3

**Comment by peterstadler** [comment]
sorry @raffazizzi , are you sure? I have v0.2.3. deployed at https://romabeta.tei-c.de and I tested this several times …
